deploy/charts/kaudit/templates/secret.yaml

```diff
@@ -120,3 +123,52 @@ spec:
 
 {{ end }}
 {{ end }}
+
+
+{{- if eq .Values.runOptions.experimental.enableEnforcement true -}}
+
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: kaudit-{{ .Values.clusterName }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: kaudit
+    app-name: kaudit-{{ .Values.clusterName }}
+webhooks:
+- name: {{ $kauditServiceName }}
+  # pass everything to kAudit admission control, except kAudit install/deploy objects
+  namespaceSelector:
+    # pass everything to kAudit admission control, except those in namespaces labeled with 'skip-kaudit-admission'  
+    # in particular - kAudit's own namespace ({{ .Values.namespace }}) should be labeled this way to skip admission  
+    matchExpressions:
```

Comment:
This should be added to the kAudit namespace (as the comment says) - I don's think it is done?

Reply:
@nitzan-n Helm 3 will not create the namespace .. so this should be part of documentation

Reply:
Also added post installation note to label the kAudit namespace